python/ml4ir/base/config/parse_args.py

```diff
@@ -401,6 +401,16 @@ def define_args(self):
             help="The number of folds for K-fold Cross Validation. Must be > 2 if testset is included in folds and > 1 otherwise.",
         )
 
+        self.add_argument(
```

Comment:
@mohazahran I recommend moving this to a model config key as it reduces code duplication a lot. Thoughts?